src/Akeneo/Pim/Enrichment/Component/Product/Exception/UnknownAttributeException.php

```diff
@@ -17,11 +21,41 @@ final class UnknownAttributeException extends PropertyException implements
     IdentifiableDomainErrorInterface,
     DocumentedErrorInterface
 {
+    /** @var DocumentationCollection */
+    private $documentation;
+
     public function __construct(string $attributeName, string $message = '', int $code = 0, \Exception $previous = null)
     {
         parent::__construct($message, $code, $previous);
 
         $this->propertyName = $attributeName;
+        $this->documentation = new DocumentationCollection([
+            new Documentation(
+                'More information about attributes: {what_is_attribute} {manage_attribute}.',
+                [
+                    new HrefMessageParameter(
+                        'What is an attribute?',
+                        'https://help.akeneo.com/pim/serenity/articles/what-is-an-attribute.html',
+                        '{what_is_attribute}'
+                    ),
+                    new HrefMessageParameter(
+                        'Manage your attributes',
+                        'https://help.akeneo.com/pim/serenity/articles/manage-your-attributes.html',
+                        '{manage_attribute}'
+                    )
```

Comment:
We can drop completely the parameter key from the Parameter objects and index them on the message directly.

Also, it feels weird (redundant) to have to give the parameter key with `{ }` if we enforce a special kind of quotation in the Documentation message:
```suggestion
                     'what_is_attribute' => new HrefMessageParameter(
                        'What is an attribute?',
                        'https://help.akeneo.com/pim/serenity/articles/what-is-an-attribute.html'
                    ),
                    'manage_attribute' => new HrefMessageParameter(
                        'Manage your attributes',
                        'https://help.akeneo.com/pim/serenity/articles/manage-your-attributes.html'
                    )
```
The alternative would be to not enforce any special quotation, then it would be logical to add them to the parameter key.